requirements.txt

```diff
@@ -0,0 +1,8 @@
+# Core dependencies
+# It is critical the VTK is *not* listed here
```

Comment:
Potential non-blocker: Would be good to know why it's not listed here.

Reply:
I added a note in the PR description about this. Should I add a note to this file directly?

> needed to make sure vtk from PyPI is not installed when using a custom VTK build variant like vtk_osmesa introduced here.

Reply:
> Should I add a note to this file directly?

Either a link to the PR or a note in the file directly. Not everyone will think to look at the blame.

Reply:
Done in 578343dc18baf71d66b2d79f185f4a8b305a3bc2